fe/fe-core/src/main/java/org/apache/doris/policy/RowPolicy.java

```diff
@@ -102,6 +102,8 @@ public class RowPolicy extends Policy implements RowFilterPolicy {
      **/
     @SerializedName(value = "originStmt")
     private String originStmt;
+    @SerializedName(value = "stmtIdx")
+    private int stmtIdx;
```

Comment:
what will stmtIdx be when do deserialization if no stmtIdx in json

Reply:
will be default 0,we only support stmtIdx is 0